### DISCUSSION TITLE
Is it possible to overwrite data with csv?

### DISCUSSION THREAD

**Discussion by GimChuang** [AntlerVC/firetable#719] [Support / Q&A] [answered]
Hi, I've been enjoying Rowy so far. Thank you so much for building this tool!

I was wondering if there is a way to overwrite data with csv.
Say i have my friend list in a firestore collection, and each document contains data of a friend.
And there is a url field (that is column in Rowy table) which is meant to be a unique url link for every friend.
I already prepared the links in a csv file. Instead of typing the links into the url field one by one, I would prefer export this table to csv, and paste the links all at once with Google Sheet or Excel. And finally import the csv back to Rowy to overwrite the documents.

I tried doing this, but it seems that Rowy doesn't export document ID. I used 'select all' for 'columns to export' when exporting, but the ID column in the exported csv is always blank. I managed to add the IDs manually and add the unique links in my exported csv, but when I tried to import it back to Rowy, it seems that there is no way to overwrite the old documents with corresponding ID, and it always creates new documents.

I was wondering if I missed something. If this is not supported for now, will it be supported in the future?

**Comment by notsidney** [comment, marked as answer]
We don’t currently support exporting IDs to CSV and importing by ID from CSV.

How did you prepare the unique URL? Is it possible to write code to generate the unique URL? If so, you can use a [Derivative column](https://docs.rowy.io/field-types/derivative).

**Reply by GimChuang** [reply]
Thank you! They are generated by codes. I will try to see if I can use Deriavative column.
